### DISCUSSION TITLE
How can I change default EBWebView folder creation behavior?

### DISCUSSION THREAD

**Discussion by vedant297** [MicrosoftEdge/WebView2Feedback#4311] [Q&A]
When we use WebView2 in Windows form application and first time run it, It could create exe-name.webview2/EBWebView folder in the executable file path. Which property I can use to avoid getting this folder created. As I don't need any functionality related to that folder.

**Comment by Paul-Williams** [comment]
See: https://github.com/MicrosoftEdge/WebView2Feedback/issues/2023#issuecomment-995071107

More detail here: https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/user-data-folder?tabs=dotnet
